tuxemon/states/pc/__init__.py

```diff
@@ -46,6 +47,7 @@
 import pygame_menu
 
 from tuxemon import graphics, prepare
+from tuxemon.animation import Animation
```

Comment:
Move this one to within `if TYPE_CHECKING` (line 62)

Reply:
Ok